digiwf-apps/packages/components/digiwf-multi-file-input/package-lock.json

```diff
@@ -7,6 +7,10 @@
     "": {
       "name": "@muenchen/digiwf-multi-file-input",
       "version": "1.10.3",
+      "dependencies": {
```

Comment:
Meines Wissens nach dürften die interen Dependencies hier nicht auftauchen, das kommt normalerweise zu Stande, wenn man `npm install` direkt in dem Repo anstatt über learna ausführt.
In de letzten Fixes war es ja auch nur das mit dem `^` unten oder?

Reply:
@simonhir 
Habe es korrgiert. Kannst du bitte nochmals prüfen?